src/app/app.component.html

```diff
@@ -448,6 +456,11 @@
                                 <span *ngIf="activeRouteResponseLastLog$ | async as activeRouteResponseLastLog" class="btn btn-link btn-icon text-primary view-body-link" (click)="goToRouteResponseLog(activeRouteResponseLastLog.UUID)">View last body sent</span>
                               </ng-container>
                             </ng-container>
+
+                            <ng-container subheading>
+                              <ng-container *ngIf="getRandomResponse()"><span class="text-warning">Activating random responses disable the rules</span></ng-container>
+                            </ng-container>
+
```

Comment:
I'm sorry, maybe what I suggested wasn't clear at all, but this message shouldn't be here which is reserved for body related messages. Instead, put it inside the "Rules" tab content at the top. I only suggested that you use the same design  than the body message. 
Something like 
```
<div class="text-warning text-center">Activating random responses disable the rules</div>
```